src/components/BulkEnrollmentPage/stepper/AddCoursesStep.jsx

```diff
@@ -20,23 +20,25 @@ const AddCoursesStep = ({
   <>
     <p>{ADD_COURSE_DESCRIPTION}</p>
     <h2>{ADD_COURSES_TITLE}</h2>
-    <SearchData>
-      <InstantSearch
-        indexName={configuration.ALGOLIA.INDEX_NAME}
-        searchClient={searchClient}
-      >
-        <Configure
-          filters={`enterprise_catalog_uuids:${subscription.enterpriseCatalogUuid}`}
-          hitsPerPage={25}
-        />
-        <SearchHeader variant="default" />
-        <CourseSearchResults
-          enterpriseId={enterpriseId}
-          enterpriseSlug={enterpriseSlug}
-          subscriptionUUID={subscription.uuid}
-        />
-      </InstantSearch>
-    </SearchData>
+    <div className="add-courses-search-data">
```

Comment:
nit: Could we use a margin utility class here to accomplish the same thing (e.g., `ml-1.5`)?

Reply:
yeah given that is the only thing I am doing with the class let me try that !

Reply:
well looks like I don't even need this class or styling anymore since the margins are now auto handled by scrollable (so this looks good even without it!)